ot/da.py

```diff
@@ -706,12 +763,19 @@ def normalizeM(self, norm):
             self.M = np.log(1 + np.log(1 + self.M))
 
 
+@deprecated("The class OTDA_sinkhorn is deprecated in 0.3.1 and will be"
```

Comment:
Love the decorator! Removed in 0.5 is short but probably a good thing since it will disappear from the examples shortly